example/package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "font-awesome": "^4.7.0",
     "ory-editor": "^0.2.24",
+    "ory-editor-plugins-infobox": "file:../plugins/infobox",
```

Comment:
Lerna should handle the linking so it doesn't have to be set absolute. To do this just set the version here to `0.0.0`, the correct one will then be resolved by lerna.
The package has to be defined in workspaces in the `package.json` of the whole repo.
(I will change this, so just fyi)